app/views/layouts/application.html.erb

```diff
@@ -16,7 +16,7 @@
     <main class="px4">
       <div class="flash-message">
         <% flash.each do |name, msg| %>
-          <%= content_tag :div, msg, class: name %>
+          <%= content_tag :div, msg.html_safe, class: name %>
```

Comment:
This has to do with view_context. According to the documentation I came I across we have to make it html_safe. Not 100 percent sure why but the link doesnt work without it. From what I gathered it has something to do with security.